src/gql/documents/mutations/event/eventFavorite.ts

```diff
@@ -0,0 +1,20 @@
+import { useMutation } from '@urql/vue'
+import { graphql } from '~~/gql/generated'
+
+export const useEventFavoriteMutation = () =>
+  useMutation(
+    graphql(`
+      mutation createEventFavorite($eventId: UUID!, $createdBy: UUID!) {
+        createEventFavorite(
+          input: { eventFavorite: { eventId: $eventId, createdBy: $createdBy } }
+        ) {
+          clientMutationId
+          eventFavorite {
+            id
+            nodeId
+            eventId
```

Comment:
Please create a reusable fragment for `EventFavorite`, see `gql/documents/fragments`, this allows for better caching.

You can read more on caching here: https://nearform.com/open-source/urql/docs/graphcache/